monkey/infection_monkey/main.py

```diff
@@ -80,12 +61,14 @@ def main():
             os.remove(log_path)
         except OSError:
             pass
-    LOG_CONFIG["handlers"]["file"]["filename"] = log_path
-    # noinspection PyUnresolvedReferences
-    LOG_CONFIG["root"]["handlers"].append("file")
 
-    logging.config.dictConfig(LOG_CONFIG)
     logger = logging.getLogger()
+    logger.setLevel(logging.DEBUG)
+
+    multiprocessing_context = get_context(method="spawn")
+    multiprocessing_queue = multiprocessing_context.Queue()
+
+    queue_listener = _get_queue_listener(multiprocessing_queue, log_path)
```

Comment:
If an exception occurs at any point before the `QueueListener` is started, the log messages will be lost.